src/app/view/components/listing-data-table/listing-data-table.component.ts

```diff
@@ -17,27 +18,41 @@ export class ListingDataTableComponent extends BaseComponent {
     @Input() columns: DataTableColumn[];
     @Input() url: string;
     @Input() actionHeaderOptions: ActionHeaderOptions;
+    @Input() buttonActions: ButtonAction[] = [];
+    @Input() sortOrder: string;
+    @Input() sortField: string;
     @Output() rowWasClicked: EventEmitter<any> = new EventEmitter;
 
     private paginatorRows: number;
     private paginatorLinks: number;
     private items: any[];
     private totalRecords: number;
     private query = '';
-    // tslint:disable-next-line:no-unused-variable
     private selectedItems = [];
+    private dateColumns: DataTableColumn[];
+    private defaultSortOrder: number;
+    private dataFormat: string;
 
     constructor(private dotcmsConfig: DotcmsConfig, private crudService: CrudService,
-    messageService: MessageService, public loggerService: LoggerService) {
+    messageService: MessageService, public loggerService: LoggerService, private formatDateService: FormatDateService) {
         super(['global-search'], messageService);
     }
 
     ngOnInit(): void {
         this.dotcmsConfig.getConfig().subscribe(configParams => {
             this.paginatorRows = configParams.paginatorRows;
             this.paginatorLinks = configParams.paginatorLinks;
-            this.loadData(this.paginatorRows, -1);
         });
+
+        this.defaultSortOrder = (this.sortOrder === 'desc' ? -1 : 1);
```

Comment:
You don't need this here... you can pass this directly in the template

```[sortOrder]="sortOrder === 'desc' ? -1 : 1"```

Because if you don't... and the ```sortOrder``` param changes you need to update the ```defaultSortOrder```.